cpp/src/Slice/CPlusPlusUtil.cpp

```diff
@@ -326,23 +326,23 @@ writeMarshalUnmarshalParams(Output& out, const ParamDeclList& params, const Oper
     bool cpp11 = (typeCtx & TypeContextCpp11) != 0;
 
     //
-    // Marshal non optional parameters.
+    // Marshal non tagged parameters.
     //
     ParamDeclList requiredParams;
-    ParamDeclList optionals;
+    ParamDeclList tagged;
```

Comment:
The tagged parameter/variable is used as a bool in most functions. Here it's a list, which is confusing. `taggedParams` would make more sense here.